exec/java-exec/src/main/java/org/apache/drill/exec/ExecConstants.java

```diff
@@ -274,6 +274,9 @@
   String ENABLE_BULK_LOAD_TABLE_LIST_KEY = "exec.enable_bulk_load_table_list";
   BooleanValidator ENABLE_BULK_LOAD_TABLE_LIST = new BooleanValidator(ENABLE_BULK_LOAD_TABLE_LIST_KEY, false);
 
+  String ANSI_QUOTES_KEY = "parser.ansi_quotes";
```

Comment:
I took that name from the old patch on this jira ticket. Completely agree with a new name. 
Done.